lib/tss2-tctildr.pc.in

```diff
@@ -7,5 +7,5 @@ Name: tss2-tctildr
 Description: Library to simplify management of TCTIs.
 URL: https://github.com/tpm2-software/tpm2-tss
 Version: @VERSION@
-Cflags: -I@includedir@
+Cflags: -I@includedir@ -I${includedir}/tss
```

Comment:
I'll submit a follow one patch to make this consistent.

Reply:
Yes, let's make them a proper autoconf variables. Thanks!